src/lib/components/datepicker/DatePicker.svelte

```diff
@@ -53,7 +53,6 @@
 	function setValue(d: Dayjs) {
 		if (!dayjs(d).isSame(value, 'day')) {
 			browseDate = clamp(d);
-			value = browseDate;
```

Comment:
Do we have any reason to have this line? It fixed the case when value is changing while scrolling months...